system_files/shared/usr/libexec/ublue-motd

```diff
@@ -23,6 +23,18 @@ if [[ -f "$TIP_FILE" ]]; then
 	fi
 
 	TIP_ESCAPED=$(escape "$TIP")
+fi
+
+# check for secure boot key
+KEY_WARN=""
+FINGERPRINT="2B:E9:91:E3:B1:B5:40:70:F4:3D:80:BB:13:EB:C6:57:E5:A3:78:0D"
```

Comment:
I'm not sure if we want to hard code the fingerprint or read it from the cert on-image?

```
openssl x509 -fingerprint -noout -in /etc/pki/akmods/certs/akmods-ublue.der
```

Reply:
this is a good suggestion.  ive implemented this in the notify-send PR